code/modules/organs/organ.dm

```diff
@@ -148,25 +148,25 @@ var/list/organ_cache = list()
 		germ_level = 0
 		return 0
 
-	var/antibiotics = owner.reagents.get_reagent_amount("spaceacillin")
+	var/antibiotics = owner.chem_effects[CE_ANTIBIOTIC] || 0
```

Comment:
`X||0 == X`

Reply:
I'm pretty sure this is a form of null check?

Reply:
Yes, in fact, strictly speaking  X||0 does not equal X if X was null

Reply:
No, X||0 when X is null will be 0 because false, rather than null, which is important in some cases like using the result in a string and you don't want it to be null and leave an empty string, but want the zero to appear instead.